centreon/src/Core/Security/Authentication/Infrastructure/Api/Login/Local/LoginController.php

```diff
@@ -58,8 +58,8 @@ public function __invoke(
             ) : null;
 
         $request = LoginRequest::createForLocal(
-            $payload["login"] ?? null,
-            $payload["password"] ?? null,
+            $payload["login"] ?: '',
+            $payload["password"] ?: '',
```

Comment:
I don't understand this change because if the key is not defined it will generate an error.
You may want to force casting ? :
```suggestion
            (string) ($payload["login"] ?? ''),
            (string) ($payload["password"] ?? ''),
```